lib/semmle-vscode-utils/package.json

```diff
@@ -1,26 +1,27 @@
 {
-  "name": "semmle-vscode-utils",
+  "name": "@github/codeql-vscode-utils",
   "description": "Shared utilities for writing Visual Studio Code extensions",
   "author": "GitHub",
-  "private": true,
-  "version": "0.0.1",
+  "private": false,
+  "version": "0.0.3",
   "publisher": "GitHub",
   "repository": {
     "type": "git",
-    "url": "https://github.com/github/vscode-codeql"
+    "url": "https://github.com/github/codeql-coreql-team.git"
```

Comment:
Should we be pointing to a private repo for a public package?

Reply:
Good question. I believe that this URL is necessary in order for GitHub Package Registry to route the published package to the right repo for permissions. The easiest thing to do is leave the private repo URL in the `package.json` as I currently have it, but I could also do the private publishing via a script that tweaks the repo URL before publishing.

Reply:
Ah...I see. It's probably fine to keep it this way. If we ever get to the point where we want to make this public, we can change. We could even publish to this repo's GPR.